charts/minio-manager/ci/test-values.yaml

```diff
@@ -15,26 +15,34 @@ users:
   - userCredentialsSecret: alice-minio-secret
     usernameSecretKey: username1
     passwordSecretKey: password1
-    policy: policy-bucket1and2
+    policy: policy-bucket1and2-get
   - userCredentialsSecret: bob-minio-secret
     usernameSecretKey: username2
     passwordSecretKey: password2
     policy: readonly
   - userCredentialsSecret: carl-minio-secret
-    policy: policy-bucket3
+    policy: policy-bucket3-delete
   - userCredentialsSecret: dave-minio-secret
 
 policies:
-  - name: policy-bucket1and2
-    readOnly: true
+  - name: policy-bucket1and2-get
     buckets:
-      - "bucket1"
-      - "bucket2"
-  - name: policy-bucket3
-    readOnly: false
+    - name: "bucket1"
+      actions:
+        - "s3:ListBucket"
+        - "s3:GetObject"
+    - name: "bucket2"
+      actions:
+        - "s3:ListBucket"
+        - "s3:GetObject"
+  - name: policy-bucket3-delete
```

Comment:
should we name the policies like so instead:
- List, Get: policy-...-read
- the above plus Put : policy-...-write
- the above plus Delete:  policy-...-fullwrite (or fullaccess)

Reply:
and as a further refinement, the chart would have these 'roles' statically defined, and with an if condition matching on the suffix as above, we can assign the correct policy actions into them.

Reply:
That could be an improvement, but I like to have flexibility to customise the policy. Can you please register it as an issue so that we can discuss it later?